<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>camAddDialog</class>
 <widget class="QDialog" name="camAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New camera</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="sbID">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Nikname</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="edSysName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLineEdit" name="edName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>MRL</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLineEdit" name="edMRL"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAccept">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonReject">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>edMRL</sender>
   <signal>textChanged(QString)</signal>
   <receiver>camAddDialog</receiver>
   <slot>onChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>camAddDialog</receiver>
   <slot>onChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edSysName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>camAddDialog</receiver>
   <slot>onChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbID</sender>
   <signal>valueChanged(int)</signal>
   <receiver>camAddDialog</receiver>
   <slot>onChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAccept</sender>
   <signal>clicked()</signal>
   <receiver>camAddDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonReject</sender>
   <signal>clicked()</signal>
   <receiver>camAddDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onChanged()</slot>
 </slots>
</ui>
